tools/report-converter/README.md

```diff
@@ -12,6 +12,7 @@ a CodeChecker server.
   * [Memory Sanitizer](#memory-sanitizer)
   * [Thread Sanitizer](#thread-sanitizer)
 * [Cppcheck](#cppcheck)
+* [Infer](#infer)
```

Comment:
I would probably use `Facebook Infer` and consider to use`fbinfer` as an abbreviation maybe for the cli argument too.
It is much easier to search for the tool with that name.